mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/StandardRenderer.java

```diff
@@ -186,7 +188,9 @@ protected void renderWaterBackground(final RenderContext renderContext) {
         for (int i = 0; i < coordinates.length; i++) {
             coordinates[i] = coordinates[i].offset(tileOrigin.x, tileOrigin.y);
         }
-        PolylineContainer way = new PolylineContainer(coordinates, renderContext.rendererJob.tile, renderContext.rendererJob.tile, Collections.singletonList(TAG_NATURAL_WATER));
+        Tag[] tags = new Tag[1];
+        tags[0] = TAG_NATURAL_WATER;
```

Comment:
Since this is a static array, we can create it once together with `TAG_NATURAL_WATER`?

Reply:
Good point, I'll improve it.